src/Models/MonitoredScheduledTask.php

```diff
@@ -28,6 +28,11 @@ class MonitoredScheduledTask extends Model
         'grace_time_in_minutes' => 'integer',
     ];
 
+    public function __construct()
```

Comment:
Hi Jonas, rather than me copying your work, might be better to put inline comments.

Should the parent construct be called here?

public function __construct()
    {
        parent::__construct();
        $this->connection = config('schedule-monitor.db_connection') ?? config('database.default');
    }

Reply:
also, the i couldn't find the connection property setup for MonitoredScheduledTaskLogItem